src/viur/core/prototypes/list.py

```diff
@@ -37,7 +37,7 @@ def viewSkel(self, *args, **kwargs) -> SkeletonInstance:
 
             :return: Returns a Skeleton instance for viewing an entry.
         """
-        return self.baseSkel(*args, **kwargs)
+        return self.skel(bones_from_request=True, **kwargs)
```

Comment:
- `*args` get lost
- `bones_from_request` cannot be conntrolled by calling `viewSkel()`? --> Please add in `viewSkel`s signature and pass it.
- 
```suggestion
        return self.skel(*args, bones_from_request=True, **kwargs)
```

Reply:
> * `*args` get lost

I want to get rid of this. It is a relic from days gone by and should be eliminated. To do it, we have to refactor all *Skel()-related stuff in prototypes, and it will also become a breaking change. Therefore, just throw it away for now here.

> * `bones_from_request` cannot be controlled by calling `viewSkel()`? --> Please add in `viewSkel`s signature and pass it.

We can do it this, do you want it to overwrite this as a default behavior?